examples/logger.py

```diff
@@ -3,13 +3,33 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
We should not change the root logger level. We only handle the log records. Changing the root logger might results in unexpected behaviors when using with other modules.